app/controllers/concerns/account_concern.rb

```diff
@@ -19,12 +19,25 @@ module AccountConcern
     # rubocop:enable Rails/LexicallyScopedActionFilter
 
     helper_method :must_reauthenticate?
+    helper_method :show_confirmation_reminder?
```

Comment:
You think this is a candidate for the Gem whenever we might get a second GOV.UK app involved?

Reply:
is this a question for me or for @barrucadu?